Travis builds are failing

### Expected Behavior

Travis builds should pass

### Actual Behavior

[They fail](https://travis-ci.org/github/LLK/scratch-paint/jobs/712282617):

```
> scratch-paint@0.2.0 lint /home/travis/build/LLK/scratch-paint

> eslint . --ext .js,.jsx

Cannot read property 'name' of null

TypeError: Cannot read property 'name' of null

    at Object.getStatelessComponent (/home/travis/build/LLK/scratch-paint/node_modules/eslint-plugin-react/lib/util/Components.js:692:45)

    at Object.getParentStatelessComponent (/home/travis/build/LLK/scratch-paint/node_modules/eslint-plugin-react/lib/util/Components.js:728:42)

    at Object.getParentComponent (/home/travis/build/LLK/scratch-paint/node_modules/eslint-plugin-react/lib/util/Components.js:623:18)

    at Object.FunctionDeclaration (/home/travis/build/LLK/scratch-paint/node_modules/eslint-plugin-react/lib/util/Components.js:904:20)

    at updatedRuleInstructions.(anonymous function) (/home/travis/build/LLK/scratch-paint/node_modules/eslint-plugin-react/lib/util/Components.js:970:43)

    at listeners.(anonymous function).forEach.listener (/home/travis/build/LLK/scratch-paint/node_modules/eslint/lib/util/safe-emitter.js:47:58)

    at Array.forEach (<anonymous>)

    at Object.emit (/home/travis/build/LLK/scratch-paint/node_modules/eslint/lib/util/safe-emitter.js:47:38)

    at NodeEventGenerator.applySelector (/home/travis/build/LLK/scratch-paint/node_modules/eslint/lib/util/node-event-generator.js:251:26)

    at NodeEventGenerator.applySelectors (/home/travis/build/LLK/scratch-paint/node_modules/eslint/lib/util/node-event-generator.js:280:22)

npm ERR! code ELIFECYCLE

npm ERR! errno 1

npm ERR! scratch-paint@0.2.0 lint: `eslint . --ext .js,.jsx`

npm ERR! Exit status 1

npm ERR! 

npm ERR! Failed at the scratch-paint@0.2.0 lint script.

npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:

npm ERR!     /home/travis/.npm/_logs/2020-07-27T17_22_56_964Z-debug.log

The command "npm run $NPM_SCRIPT" exited with 1.
```

--- a/package.json
+++ b/package.json
@@ -64,7 +64,7 @@
     "eslint-config-import": "^0.13.0",
     "eslint-config-scratch": "^5.0.0",
     "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-react": "^7.2.1",
+    "eslint-plugin-react": "7.20.3",
     "gh-pages": "github:rschamp/gh-pages#publish-branch-to-subfolder",
     "html-webpack-plugin": "3.2.0",
     "jest": "^22.2.2",

